flask_ask/models.py

```diff
@@ -209,11 +268,11 @@ def _audio_item(self, stream_url=None, offset=0, push_buffer=True):
         # new stream
         else:
             stream['url'] = stream_url
-            stream['token'] = str(uuid.uuid4())
+            stream['token'] = str(random.randint(10000, 100000))
```

Comment:
if the token must be unique, this no longer ensures it. i don't know much about the stream handling code, but this would appear to be a breaking change if uniqueness was required originally. thoughts?